package/libgluonutil/src/libgluonutil.h

```diff
@@ -26,11 +26,14 @@
 
 #ifndef _LIBGLUON_LIBGLUON_H_
 #define _LIBGLUON_LIBGLUON_H_
+#include <arpa/inet.h>
```

Comment:
This should include `netinet/in.h` for `struct in6_addr`, `arpa/inet.h` should be moved to the main file.

Reply:
done. What is the reasoning behind moving arpa/inet.h to the main file?

Reply:
One should always try to use the "smallest" include, especially in public header files, to avoid forcing others to compile more code than necessary when they use your headers. The header file only needs `struct in6_addr`, which is defined in `netinet/in.h` according to the docs.